src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
```

Comment:
Ensure that the element with the class 'logo' exists in the DOM. If it doesn't, `document.querySelector('.logo')` will return `null`, and attempting to add an event listener to `null` will result in an error.